tests/e2e/conftest.py

```diff
@@ -52,26 +53,29 @@ def e2e_test_secrets(request: FixtureRequest) -> EndToEndTestSecrets:
     raise ValueError(f"Unknown e2e_env: {e2e_env}.")
 
 
+@pytest.fixture
+def email(request: FixtureRequest) -> str:
+    email_env = os.getenv("E2E_GOVUK_NOTIFY_AUTHENTICATE_AS")
+    email_option = request.config.getoption("authenticate_as")
+    email = email_option or email_env
+
+    if not email:
+        raise ValueError("Set E2E_GOVUK_NOTIFY_AUTHENTICATE_AS in the environment or use --authenticate_as")
```

Comment:
Controversial: if we're going this route, just put the email address in here as a default value? Yes we'd be leaking it, but does that matter (genuine question, not dismissive)? Would rather as few things as possible need configuring/specifying to run the e2e tests.